libraries/AP_EFI/AP_EFI_Currawong_ECU.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * AP_EFI_Currawong_ECU.cpp
+ *
+ *      Author: Reilly Callaway / Currawong Engineering Pty Ltd
+ */
+
+#include "AP_EFI_Currawong_ECU.h"
+
+#if HAL_EFI_CURRAWONG_ECU_ENABLED
+
+#include <AP_Param/AP_Param.h>
+#include <AP_PiccoloCAN/piccolo_protocol/ECUPackets.h>
+#include <AP_Math/definitions.h>
+
+#define KGPM3_TO_GPCM3(x) (0.001f * x)
```

Comment:
Lets add this into here instead? https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Math/definitions.h

`#define KG_PER_M3_TO_G_PER_M3 (x) (0.001 * x)`    // kg/m^3 to g/m^3

Reply:
Done! Small note that it is kg/m^3 to g/**c**m^3, not g/m^3. Which is why we get a factor of 0.001 instead of 1000.

Reply:
Lol yeah.. I was looking at other things and typing :)

Feel free to resolve conversations that you have fixed.